src/lib.rs

```diff
@@ -85,12 +85,16 @@ fn get_class(conn: &xcb::Connection, id: u32, options: &Options) -> Result<Strin
 
     let result = String::from_utf8(buf)?;
     let mut results = result.split('\0');
-    results.next_back();
+    let wm_instance = results.next().unwrap();
```

Comment:
Try not to use unwrap in anything but test code, use the [match construct](https://doc.rust-lang.org/rust-by-example/flow_control/match.html).

Reply:
I tried to fix this in latest commit.